src/components/Day/DayInfoSection/DayInfo.tsx

```diff
@@ -1,11 +1,10 @@
 import React from 'react';
-import { useRecoilValue } from 'recoil';
-import { dayInfo } from 'src/states';
 import { theme } from 'src/styles/theme';
+import { getTodayDate } from 'src/utils/getDate';
 import styled from 'styled-components';
 
 function DayInfo() {
-  const today = useRecoilValue(dayInfo);
+  const today = getTodayDate(0);
```

Comment:
위에서 말한 `const DEFAULT_DATE_CHANGE = 0;` 를 전역 상수 파일로 빼내고, 여기서도 임포트 해서 
```suggestion
  const today = getTodayDate(DEFAULT_DATE_CHANGE);
```
요렇게 사용해도 좋을 것 같아요!